src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/LicenseScanTests.cs

```diff
@@ -150,7 +150,7 @@ public void ScanForLicenses()
         string ignoreOptions = string.Join(" ", s_ignoredFilePatterns.Select(pattern => $"--ignore {pattern}"));
         ExecuteHelper.ExecuteProcessValidateExitCode(
             "scancode",
-            $"--license --processes 4 --strip-root --only-findings {ignoreOptions} --json-pp {scancodeResultsPath} {Config.LicenseScanPath}",
+            $"--license --processes 4 --timeout 240 --strip-root --only-findings {ignoreOptions} --json-pp {scancodeResultsPath} {Config.LicenseScanPath}",
```

Comment:
For consistency, can you follow the same pattern that exists in main by defining this as a constant?

https://github.com/dotnet/sdk/blob/60e06cc3059bd58e88ec62c878d256ad25b6d568/src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/LicenseScanTests.cs#L134-L143

Reply:
Sure thing!